.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Spock Gitter Chat
+    url: https://gitter.im/spockframework/spock
+    about: Please ask and answer questions here.
+  - name: Stack Overflow
+    url: https://stackoverflow.com/questions/tagged/spock
+    about: For questions of the type "why isn't this working".
```

Comment:
I said this before, but somehow GitHub seems to have swallowed that one.
I hope it is the only one missing.
I'd remove the dots from the `about`s here.
Or alternatively (I prefer removal) add dots to the template `about`s.
It looks more uniform then:
![grafik](https://user-images.githubusercontent.com/325196/119428813-e50d2300-bd0d-11eb-98a4-f0f0aeaa6adb.png)